docs/self-hosted/kubernetes-helm.md

```diff
@@ -0,0 +1,100 @@
+# Installing the Private Packagist Self-Hosted Helm chart in an existing Kubernetes cluster
+##
+
+The Private Packagist Self-Hosted Helm chart allows you to install Private Packagist in an existing Kubernetes cluster,
+to instead install Private Packagist Self-Hosted without an existing Kubernetes cluster follow [this guide](./kubernetes-embedded.md).
+
+## General requirements
+
+1. A Kubernetes cluster v1.23 or v1.24
+1. Your username and password to log in to the registry. Don't have one yet? [Sign up for a free trial license!](https://packagist.com/self-hosted)
+1. One (sub-)domain to operate the web interface, e.g. packagist.myintranet.com
+1. One (sub-)domain to operate the composer repository, e.g. repo.packagist.myintranet.com or packagist-repo.myintranet.com
+1. An SSL certificate valid for both chosen domains
+1. An SMTP server or a GMail account for Private Packagist Self-Hosted to send email
+1. If your firewall restricts external connections the following domains must be accessible from the server:
+  * hub.docker.com
+  * proxy.replicated.com
+  * registry.replicated.com
+  * replicated.app
+  * amazonaws.com
+  * k8s.gcr.io
+
+## Installation
+
+Private Packagist Self-Hosted requires PostgreSQL, Redis, and blob storage to store application data and Composer packages.
+You can either use the build-in options to come with the Helm chart or use your own PostgreSQL, Redis, and blob storage.
+For blob storage, we currently support Azure Blob Storage, Google Cloud Storage, AWS S3, and other S3-compatible storage solutions.
+
+Please note that if you chose to use the built-int solution then each of the storage requires one or more volumes using
+[dynamic volume provision](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/) to allocate storage for the different Pods.
+Configure the Storage Class in global values section.
+
+### Annotated configuration
+
+HELM_CHART_VALUES_FILE
+
+#### Login to the Helm registry and install the chart
+
+To install the Private Packagist Self-Hosted Helm Chart configure values based on your setup and then run the commands below.
+Make sure you replace `YOUR_USERNAME`, `YOUR_PASSWORD`, `YOURVALUES.yaml`, and `VERSION` with your values before running the commands.
+
+```
+helm registry login registry.replicated.com --username YOUR_USERNAME --password YOUR_PASSWORD
+helm install -f YOURVALUES.yaml private-packagist oci://registry.replicated.com/privatepackagistkots/unstable/private-packagist --version VERSION
+```
+
```

Comment:
add a small paragraph how to check if it's running and how to access it?

Reply:
Added